where_to_get_help.Rmd

```diff
@@ -0,0 +1,44 @@
+# Where to get help 
+
+If you are working on R it is possible that you will come across an issue where you would need some assistance to solve it. If you require help, there are options available to seek assistance or get some feedback which are discussed in this chapter. If the question involves process or tool usage then please check the rest of this guide first as it should answer your question. Please be polite, patient, and respectful of people's time while you ask for feedback or help. 
+
+## Mailing Lists
+
+There are quite a few [mailing lists](https://www.r-project.org/mail.html) for getting help with R:
+
+  * [R-help](https://stat.ethz.ch/mailman/listinfo/r-help): 
+      * Discussions about problems and solutions using R.
+      * Announcements about the development of R.
+      * Announcements about the availability of new code.

```

Comment:
I'm not following this list but on the https://www.r-project.org/mail.html these two points are not listed. I haven't checked the mail archive, perhaps it is done regularly and I missed this is the right place to announce these kind of things.

Reply:
Oh, thanks